mono/utils/Makefile.am

```diff
@@ -239,8 +241,8 @@ endif
 
 libmonoutils_la_SOURCES = $(monoutils_sources) $(arch_sources)
 
-libmonoutils_il2cpp_la_SOURCES = $(libmonoutils_la_SOURCES)
-libmonoutils_il2cpp_la_CFLAGS = -DIL2CPP_ON_MONO -DDISABLE_JIT -DDISABLE_REMOTING -fexceptions
+libmonoutils_unity_il2cpp_la_SOURCES = $(libmonoutils_la_SOURCES) $(unity_files)
+libmonoutils_unity_il2cpp_la_CFLAGS = -DIL2CPP_ON_MONO -DDISABLE_JIT -DDISABLE_REMOTING -fexceptions -DPLATFORM_UNITY -I../../../../il2cpp/build/libil2cpp/os/c-api
```

Comment:
This is the way that I did it for eglib as well. Is it possible to set CFLAGS for a specific target in the configure.ac file? I thought that it was not.